### PR TITLE
v1.1.8

### PR DESCRIPTION
**Pull request by dr-dimitru**
- Remove logging
- Minor docs update

### CODE REVIEW

File: spiderable_server.js [hunk 1]
```diff
@@ -76,7 +76,6 @@ var PHANTOM_SCRIPT = Assets.getText("phantom_script.js");
 WebApp.connectHandlers.use(function (req, res, next) {
   // _escaped_fragment_ comes from Google's AJAX crawling spec:
   // https://developers.google.com/webmasters/ajax-crawling/docs/specification
-  console.log (req.headers);
```
Comment (jazeee) [inline spiderable_server.js:79]: Oops, sorry about that. I thought I took that out.

File: README.md [hunk 2]
```diff
@@ -66,8 +66,7 @@ You will need to set `Meteor.isReadyForSpiderable` to `true` when your route is
 I am deprecating `Meteor.isRouteComplete=true`, but it will work until at least 2015-12-31 after which I'll remove it...
 See [code for details](https://github.com/jazeee/jazeee-meteor-spiderable/blob/master/phantom_script.js)
 
-If using IronRouter, I recommend that you create a base controller with `onAfterAction` function. Once checking for @isReady() You can set `Meteor.isReadyForSpiderable = true` in that.
-
+If using IronRouter, I recommend that you create a base controller with `onAfterAction` function. Once checking for `@isReady()`, you can set `Meteor.isReadyForSpiderable = true` in that.
```
Comment (jazeee) [inline README.md:69]: Actually, it should be @ready(), which I will also fix.

**Comment by jazeee** [comment]
Sorry about this issue...
I also updated the documentation with more details such as testing tips that I learned.

Really important: Make sure to read about the `Meteor.ready()` stuff, since it was messing things up for me.

If you use Iron router, you want to ensure that you don't mark as ready for spiderable until `Meteor.ready()`

File: README.md (hunk 2 above)
Comment (dr-dimitru) [inline README.md:69]: @jazeee Should I send pull request or you will fix it?

**Comment by dr-dimitru** [comment]
@jazeee ok, I see you've already fixed text at readme.
Thank you for accepting PR. Sometimes it happens, when developers forget about logging

**Pull request merged**